cli/commands/cluster/member-update.go

```diff
@@ -33,7 +33,7 @@ func MemberUpdateCommand(cli *cli.SensuCli) *cobra.Command {
 				return fmt.Errorf("error updating cluster member: %s", err)
 			}
 
-			fmt.Fprintf(cmd.OutOrStdout(), "Updated member with ID %x in cluster\n", id)
+			fmt.Fprintf(cmd.OutOrStdout(), "OK")
```

Comment:
I know you mentioned you were unsure about the cluster commands. I think it's safe to leave those as is, for now.
```suggestion
 			fmt.Fprintf(cmd.OutOrStdout(), "Updated member with ID %x in cluster\n", id)
```